api/metrics-api-jaxrs/src/main/java/org/hawkular/metrics/api/jaxrs/MetricHandler.java

```diff
@@ -824,6 +925,7 @@ public void updateCounter(@Suspended final AsyncResponse asyncResponse, @PathPar
     }
 
     @POST
+    @ApiOperation(value = "Update value of a counter", hidden = true)
```

Comment:
hidden=true, I guess it's when you want to hide a method which is not stable yet?